services/frontend/src/components/layout/AppBar.vue

```diff
@@ -23,35 +31,98 @@
       class="vertical-center"
     ></v-autocomplete>
 
-    <v-btn icon @click="goToSearch">
-      <v-icon>mdi-book-search-outline</v-icon>
-    </v-btn>
-
+    <v-tooltip bottom>
+      <template v-slot:activator="{ on, attrs }">
+        <v-btn 
+          icon 
+          @click="goToSearch"
+          v-bind="attrs"
+          v-on="on">
+          <v-icon>mdi-book-search-outline</v-icon>
+        </v-btn>
+      </template>
+      <span>Search</span>
+    </v-tooltip>
+    
     <v-spacer></v-spacer>
 
-    <v-btn icon >
-      <v-icon>mdi-food-variant</v-icon>
-    </v-btn>
+    <v-tooltip bottom>
+      <template v-slot:activator="{ on, attrs }">
+        <v-btn 
+          icon 
+          @click="goToSearch"
+          v-bind="attrs"
+          v-on="on">
+            <v-icon>mdi-food-variant</v-icon>
+        </v-btn>
+      </template>
+      <span>Your Recpies</span>
+    </v-tooltip>
+    
 
-    <v-btn icon>
-      <v-icon>mdi-chef-hat</v-icon>
-    </v-btn>
+    <v-tooltip bottom>
+      <template v-slot:activator="{ on, attrs }">
+        <v-btn 
+          icon 
+          @click="goToAddRecipe"
+          v-bind="attrs"
+          v-on="on">
+          <v-icon>mdi-chef-hat</v-icon>
+        </v-btn>
+      </template>
+      <span>Add a recpie</span>
+    </v-tooltip>
 
     <v-divider vertical></v-divider>
 
-      <v-btn v-if="isLoggedIn" icon @click="logout">
-        <v-icon>mdi-logout</v-icon>
-      </v-btn>
-
-      <v-avatar v-if="isLoggedIn" @click="goToAccount" color="primary" size="56">PW</v-avatar>
-
-      <v-btn v-if="!isLoggedIn" icon @click="login">
-        <v-icon>mdi-login</v-icon>
-      </v-btn>
-
-      <v-btn v-if="!isLoggedIn" icon @click="register">
-        <v-icon>mdi-account-plus-outline</v-icon>
-      </v-btn>
+      <v-tooltip bottom>
+      <template v-slot:activator="{ on, attrs }">
+        <v-btn 
+          v-if="isLoggedIn"
+          icon 
+          @click="logout"
+          v-bind="attrs"
+          v-on="on">
+          <v-icon>mdi-logout</v-icon>
+        </v-btn>
+      </template>
+      <span>Logout</span>
+    </v-tooltip>
+
+    <v-tooltip bottom>
+      <template v-slot:activator="{ on, attrs }">
+        <v-avatar v-on="on" v-bind="attrs" v-if="isLoggedIn" @click="goToAccount" color="primary" size="56">PW</v-avatar>
```

Comment:
I think we should consider putting here actual initials of the user's name and a random color (assigned during account creation).